i18n/zh/docusaurus-plugin-content-docs/current/vm/create-vm.md

```diff
@@ -0,0 +1,181 @@
+---
+sidebar_position: 1
+sidebar_label: 创建虚拟机
+title: ""
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - 虚拟机
+  - VM
+  - 创建虚拟机
+Description: 从"虚拟机"页面创建一个或多个虚拟机。
+---
+
+# 创建虚拟机
+
+## 如何创建虚拟机
+
+你可以从 **Virtual Machines** 页面创建一个或多个虚拟机。
+
+:::note
+
+请参阅[此页面](./create-windows-vm.md)创建 Windows 虚拟机。
+
+:::
+
+1. 选择创建单个实例或多个实例。
+1. 选择 VM 的命名空间，只有 `harvester-public` 命名空间对所有用户可见。
+1. 输入虚拟机名称（必填）。
+1. （可选）VM 模板是可选的，你可以选择 `iso-image`、`raw-image` 或 `windows-iso-image` 模板来快速创建虚拟机实例。
+1. 配置虚拟机的 CPU 和内存（如果要超额配置，请参阅 [overcommit-config](../advanced/settings.md#overcommit-config)）。
+1. 选择 SSH 密钥或上传新密钥。
+1. 在 **Volumes** 选项卡上选择自定义虚拟机镜像卷。默认磁盘将是根磁盘。你可以向虚拟机添加更多磁盘。
```

Comment:
Hailong and I discussed it before, and we think it will be better if we use the original English text here. Because we have screenshots in the document, it will be easier for users to follow the instructions if we keep the text and screenshots consistent.

Reply:
Yes, it may be better if there is a Chinese screenshot. When the user switches to Chinese, the Chinese screenshot will be displayed (but this should not be a problem. You can ask @guangbochen  whether it needs a Chinese screenshot)

Reply:
Just discussed with Guangbo, and we might still use English screenshots and English text for the time being.